package.json

```diff
@@ -3,18 +3,22 @@
   "version": "0.0.1",
   "description": "Mono repo for playing the game Connect Four as CLI or SPA.",
   "scripts": {
-    "build": "npm run build --workspace connect-four-reboot-cli && npm run build --workspace connect-four-reboot-spa",
+    "build-cli": "npm run build --workspace connect-four-reboot-cli",
+    "build-spa": "npm run build --workspace connect-four-reboot-spa",
+    "build-api": "npm run build:ts --workspace connect-four-reboot-api",
+    "build": "npm run build-cli && npm run build-spa && npm run build-api",
     "dev-cli": "npm run dev --workspace connect-four-reboot-cli",
     "dev-spa": "npm run dev --workspace connect-four-reboot-spa",
-    "test": "vitest run",
-    "format": "prettier --write 'packages/**/*.{ts,tsx,js,json,scss,md}'",
-    "clear": "rm -Rf node_modules apps/cli/dist apps/spa/dist apps/spa/node_modules apps/cli/node_modules",
-    "type-check": "npx tsc --noEmit --project ./apps/cli && npx tsc --noEmit --project ./apps/spa"
+    "dev-api": "npm run dev --workspace connect-four-reboot-api",
+    "test": "vitest run && node --test",
+    "clear": "rm -rf node_modules && npm run clear --workspace connect-four-reboot-spa && npm run clear --workspace connect-four-reboot-cli && npm run clear --workspace connect-four-reboot-api",
```

Comment:
use the `--workspaces` to run in all workspaces instead of defining them one by one. 

https://docs.npmjs.com/cli/v7/using-npm/workspaces?v=true#running-commands-in-the-context-of-workspaces